browser/src/Services/Explorer/ExplorerView.tsx

```diff
@@ -269,11 +266,77 @@ export interface IExplorerViewProps extends IExplorerViewContainerProps {
     updated: string[]
 }
 
-import { SidebarEmptyPaneView } from "./../../UI/components/SidebarEmptyPaneView"
+interface ISneakableNode extends IExplorerViewProps {
+    node: Node
+    selectedId: string
+}
 
-import { commandManager } from "./../CommandManager"
+const SneakableNode = ({ node, selectedId, ...props }: ISneakableNode) => {
+    const handleClick = () => {
+        props.onClick(node.id)
+    }
+
+    return (
+        <Sneakable callback={handleClick}>
+            <NodeView
+                node={node}
+                isSelected={node.id === selectedId}
+                isCreating={props.isCreating}
+                onCancelCreate={props.onCancelCreate}
+                onCompleteCreate={props.onCompleteCreate}
+                onCompleteRename={props.onCompleteRename}
+                isRenaming={props.isRenaming}
+                onCancelRename={props.onCancelRename}
+                updated={props.updated}
+                yanked={props.yanked}
+                moveFileOrFolder={props.moveFileOrFolder}
+                onClick={handleClick}
+            />
+        </Sneakable>
+    )
+}
+
+const ExplorerContainer = styled.div`
+    height: 100%;
+    ${enableMouse};
+`
 
 export class ExplorerView extends React.PureComponent<IExplorerViewProps, {}> {
+    private _virtualizedList: List
+
+    public componentDidUpdate(prevProps: IExplorerViewProps) {
+        if (
+            prevProps.isCreating !== this.props.isCreating ||
+            prevProps.isRenaming !== this.props.isRenaming
+        ) {
+            this._virtualizedList.recomputeRowHeights()
+            this._virtualizedList.forceUpdate()
+        }
+    }
+
+    public calculateHeight = (selectedId: string) => ({ index }: { index: number }) => {
+        const { nodes, isCreating, isRenaming } = this.props
+        const node = nodes[index]
+        const { id, name } = node
+        const isSelected = id === selectedId
+
+        const PADDING = 10
+        const fontSize = parseInt(this.props.fontSize, 10)
```

Comment:
Hmm, I wonder if this handles the various cases we allow for fontsize? For example, it looks like `11px`, `11pt`, `11em` would all get handled the same way here, but they would actually have very different _pixel_ sizes. I think what we'd want instead is the measured font size, or some more deterministic way to get the size of the cells.

Reply:
@bryphe I used the menu component which has a static value of `20` as my initial reference point for implementing this and I haven't noted any issue with the value we pass in there, I also tried various sizes with a static value for `rowHeight` locally and didn't get any issues re. different units I can look at converting em to px as for pt I really hope ppl aren't giving units in pt as that'll be very problematic on multiple fronts.